app/views/users/_vitals.html.erb

```diff
@@ -3,22 +3,21 @@
 </div>
 
 <div class="list-group-item">
```

Comment:
Not 100% sure, but I believe that list-group-items need to be part of a list group.